src/utils/__tests__/style-formatter.spec.js

```diff
@@ -268,6 +271,31 @@ describe('style-formatter', () => {
       expect(button.children[0].style.fontSize).toEqual('9.200000000000001px');
     });
 
+    it('should use default fixed font size when size behavior is fixed', () => {
```

Comment:
actually in this case the font value will be 20 in default which will get assigned to `button.children[0].style.fontSize` that this test is checking for. 

So `sizeFixed` will not be undefined when getting it from `layout.style.font.sizeFixed`

I logged it's value and figured out that it return 20 by default and i remember Niek sets the default value for it in here: https://github.com/qlik-oss/sn-action-button/blob/9079129b19e43509e4dceb3bb94373482aefd22a/src/object-properties.js#L68 which points to here: https://github.com/qlik-oss/sn-action-button/blob/9079129b19e43509e4dceb3bb94373482aefd22a/src/style-defaults.js#L4 -> which is 20. 

so it will not get undefined i think? or maybe I'm wrong somehow 🤨

Reply:
nope you are correct!